src/wasm-linker.h

```diff
@@ -259,6 +271,9 @@ class Linker {
   // Returns false if an error occurred.
   bool linkArchive(Archive& archive);
 
+  Address getStackPointerAddress() const;
+  Address staticBump() const;
```

Comment:
should this be called `getStaticBump` for consistency? Also probably worth saying what exactly the static bump is here.

Reply:
Yes it should be.